### ISSUE TITLE
Query for all users

### ISSUE THREAD

**Issue by drimpact** [impactasaurus/server#197]
If a user has the "user:read" scope, they can read all users in the system. Blocked by #196 and #171

**Comment by drimpact** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Newsletter Support](https://trello.com/c/og3JgINj/65-newsletter-support)

**Comment by drimpact** [comment]
No longer needed by #199, will close until it is needed again
